checker/tests/nullness/ParameterExpression.java

```diff
@@ -46,33 +46,35 @@ public void m5() {
 
     @EnsuresNonNull("param")
     // :: error: (flowexpr.parse.error)
-    // :: warning: (expression.parameter.name.invalid)
     public void m6a(Object param) {
         param = new Object();
     }
 
     @EnsuresNonNull("param")
     // :: error: (flowexpr.parse.error)
-    // :: warning: (expression.parameter.name.invalid)
     public void m6b(Object param) {
         // :: error: (assignment.type.incompatible)
         param = null;
     }
 
     @EnsuresNonNull("param")
     // :: error: (flowexpr.parse.error)
-    // :: warning: (expression.parameter.name.invalid)
     public void m6c(@Nullable Object param) {
         param = new Object();
     }
 
     @EnsuresNonNull("param")
     // :: error: (flowexpr.parse.error)
-    // :: warning: (expression.parameter.name.invalid)
     public void m6d(@Nullable Object param) {
         param = null;
     }
 
+    @EnsuresNonNull("param.toString()")
+    // :: error: (flowexpr.parse.error)
```

Comment:
The full error is:

ParameterExpression.java:74: error: [flowexpr.parse.error] cannot parse the expression 'param.toString()' in the postcondition EnsuresNonNull on the declaration of method 'm6e': Invalid 'param' because Use "#1" rather than "param"